docs/ManualDeploymentGuideScalarDBClusterOnEKS.md

```diff
@@ -0,0 +1,59 @@
+# Deploy ScalarDB Cluster on Amazon Elastic Kubernetes Service (EKS)
+
+This guide explains how to deploy ScalarDB Cluster on Amazon Elastic Kubernetes Service (EKS).
+
+In this guide, you will create one of the following two environments in your AWS environment. The difference between the two environments is which [client modes](https://github.com/scalar-labs/scalardb-cluster/blob/main/docs/developer-guide-for-scalardb-cluster-with-java-api.md#client-modes) you use:
+
+* Use [`direct-kubernetes` client mode](https://github.com/scalar-labs/scalardb-cluster/blob/main/docs/developer-guide-for-scalardb-cluster-with-java-api.md#direct-kubernetes-client-mode). In this mode, you deploy your application in the same EKS cluster as your ScalarDB Cluster deployment.
+
+  ![image](./images/png/EKS_ScalarDB_Cluster_Direct_Kubernetes_Mode.drawio.png)  
+
+* Use [`indirect` client mode](https://github.com/scalar-labs/scalardb-cluster/blob/main/docs/developer-guide-for-scalardb-cluster-with-java-api.md#indirect-client-mode). In this mode, you deploy your application in an environment that is different from the EKS cluster that contains your ScalarDB Cluster deployment.
+
+  ![image](./images/png/EKS_ScalarDB_Cluster_Indirect_Mode.drawio.png)
+
+## Step 1. Subscribe to ScalarDB Cluster in AWS Marketplace
+
+You must get the ScalarDB Cluster container image by visiting [AWS Marketplace](https://aws.amazon.com/marketplace/pp/prodview-rzbuhxgvqf4d2) and subscribing to ScalarDB Cluster. For details on how to subscribe to ScalarDB Cluster in AWS Marketplace, see [Subscribe to Scalar products from AWS Marketplace](./AwsMarketplaceGuide.md#subscribe-to-scalar-products-from-aws-marketplace).
```

Comment:
When I go to the [AWS Marketplace](https://aws.amazon.com/marketplace/pp/prodview-rzbuhxgvqf4d2) link, I see a page that shows that the latest version is `ScalarDB Server 3.9`. I worry this might be confusing for users if they are expecting to see `ScalarDB Cluster`.

Reply:
Ah, sorry. I overlooked this part.
We have to update the URL since the `ScalarDB (BYOL)` and `ScalarDB Cluster [Standard|Premium] Edition` are different products in AWS Marketplace.
I updated the URL in c10051a9403bf875578bad7402f581318b49d1d1.

Reply:
Thanks! I can't confirm the contents of the AWS Marketplace page when I'm not logged in, but I think that might be intentional.

Reply:
Got it. Thanks!